blog/views.py

```diff
@@ -0,0 +1,12 @@
+from django.shortcuts import render
+from .models import Post
+# Create your views here.
+
+def home(request):
+    context = {
+        'posts': Post.objects.all()
+    }
+    return render(request, 'blog/home.html', context=context)
+
+def about(request):
+    return render(request, 'blog/about.html', {'title': 'About'})
```

Comment:
defined the context first and then put it in the render

Reply:
done.. increases readability